pysc2/pysc2/lib/features.js

```diff
@@ -1060,34 +1051,34 @@ class Features {
           "position.`)
     }
     map_size = point.Point.build(map_size)
-    this._world_to_world_tl = transform.Linear(
-      point.Point(1, -1),
-      point.Point(0, map_size.y)
+    this._world_to_world_tl = new transform.Linear(
+      new point.Point(1, -1),
+      new point.Point(0, map_size.y)
     )
-    this._world_tl_to_world_camera_rel = transform.Linear(/*offset=*/ -(map_size / 4))
+    const offset = -map_size.div(4)
+    this._world_tl_to_world_camera_rel = new transform.Linear(null, offset)
     if (feature_dimensions) {
-      const world_camera_rel_to_feature_screen = transform.Linear(
-        feature_dimensions.screen / camera_width_world_units,
-        feature_dimensions.screen / 2
+      const world_camera_rel_to_feature_screen = new transform.Linear(
+        feature_dimensions.screen.div(camera_width_world_units),
+        feature_dimensions.screen.div(2)
       )
-      this._world_to_feature_screen_px = transform.Chain(
+      this._world_to_feature_screen_px = new transform.Chain(
         this._world_to_world_tl,
         this._world_tl_to_world_camera_rel,
         world_camera_rel_to_feature_screen,
-        transform.PixelToCoord()
+        new transform.PixelToCoord()
       )
     }
-    // If we don't have a specified raw resolution, we do no transform.
-    const world_tl_to_feature_minimap = transform.Linear(
-      raw_resolution ? /*scale=*/raw_resolution / map_size.max_dim() : null
-    )
-    this._world_to_minimap_px = transform.Chain(
+    // If we don't have a specified raw resolution, we do no new transform.
+    const scale = raw_resolution ? raw_resolution.div(map_size.max_dim()) : null
+    const world_tl_to_feature_minimap = new transform.Linear(scale)
+    this._world_to_minimap_px = new transform.Chain(
       this._world_to_world_tl,
       world_tl_to_feature_minimap,
-      transform.PixelToCoord()
+      new transform.PixelToCoord()
     )
     this._camera_size = (
-      raw_resolution / (map_size.max_dim() * camera_width_world_units)
+      raw_resolution.div(map_size.max_dim() * (camera_width_world_units))
```

Comment:
It should be 
raw_resolution.div(map_size.max_dim())* (camera_width_world_units)
original: 
raw_resolution / map_size.max_dim() * camera_width_world_units)